related-tooling.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Related Tooling
+sidebar: home_sidebar
+permalink: /related-tooling/
+toc: false
+---
+
+Open Source tooling, such as for managing the state of Pulp and integrating it with other projects, that is either maintained by the commnunity or other projects, or that is developed by the Pulp Project but not yet in production.
+
+## Foreman / Katello
+
+[Foreman](https://theforeman.org/) is a complete lifecycle management tool for physical and virtual servers. [Katello](https://theforeman.org/plugins/katello/) brings the full power of content management alongside the provisioning and configuration capabilities of Foreman.
+
+Foreman & Katello use Pulp for managing repositories software packages. It provides a graphical WebUI for Pulp and integrates it with its workflows, but does not expose every underlying feature & content type of Pulp.
+
+Foreman & Katello are currently in the incremental [process of migrating from Pulp 2 to Pulp 3.](https://community.theforeman.org/t/update-on-katello-pulp3-integration-and-upcoming-roadmap/15899)
+
+## Ansible Modules for Pulp aka Squeezer
+
+[This project](https://github.com/mdellweg/ansible_modules_pulp) provides ansible modules (currently only for pulp-file) to control a pulp3 server in a descriptive way. This is neither to be confused with ansible-pulp to install pulp, nor pulp_ansible to manage ansible content in pulp. It has been wildly inspired by the sister project [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules).
+
+## pulp_rpm_repos
+
+pulp_rpm_repos ([GitHub](https://github.com/juan-cabrera/pulp_rpm_repos), [Galaxy](https://galaxy.ansible.com/juan_cabrera/pulp_rpm_repos)): Use pulp API client to manage repositories in a pulp server
+
+This Ansible role interacts with a Pulp 3 server. It helps to create and manage rpm repositories.
```

Comment:
This is not specific to your PR, as we all could do better, so if you'd prefer to open an additional issue, I understand, but the interchangeable use of "pulp" and "Pulp" should probably be replaced with one consistent use.  (In the first line of the pulp_rpm_repos section, it's "pulp server" and in the following line, it's "Pulp 3 server".

Reply:
I agree and will do a find and replace with "Pulp", like our official docs use. I personally wrote "Pulp", but I copied and pasted these descriptions with as few changes as possible.

Perhaps I should use markdown quote blocks instead?

Reply:
If you are wholesale taking descriptions from elsewhere, sure, quote blocks would make sense.  Also ok with a find replace, your choice.